src/app-layout/visual-refresh/main.tsx

```diff
@@ -61,7 +62,7 @@ export default function Main() {
           `var(${customCssProps.mobileBarHeight})`,
           !!disableBodyScroll,
           isMobile,
-          hasBackgroundOverlap && !isMobile ? tokens.spaceScaledS : '0px'
+          hasBackgroundOverlap && hasStickyBackground && !isMobile ? tokens.spaceScaledS : '0px'
```

Comment:
`tokens.spaceScaledS` here defines the sticky position, but it is only needed when rendering a full-page table, otherwise it should be 0. Otherwise the same space is already applied by defining [this padding](https://github.com/cloudscape-design/components/blob/main/src/table/styles.scss#L185) inside the header instead of outside.

If adding this space without the sticky background present, the unexpected space will appear and the content will be visible behind.